scheduler/src/test/java/io/kestra/scheduler/SchedulerScheduleTest.java

```diff
@@ -527,6 +527,62 @@ void failedEvaluationTest() {
             // wait for execution
             Flux<Execution> receive = TestsUtils.receive(executionQueue, either -> {
                 Execution execution = either.getLeft();
+                assertThat(execution).isNotNull();
+                assertThat(execution.getFlowId()).isEqualTo(flow.getId());
+                assertThat(execution.getState().getCurrent()).isEqualTo(State.Type.FAILED);
+
+                queueCount.countDown();
+            });
+
+            scheduler.run();
+
+            queueCount.await(1, TimeUnit.MINUTES);
+            // needed for RetryingTest to work since there is no context cleaning between method => we have to clear assertion receiver manually
+            receive.blockLast();
+
+            assertThat(queueCount.getCount()).isEqualTo(0L);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+    // covers trigger evaluation failures propagated to evaluateScheduleTrigger() such as invalid expressions, input resolving issues at scheduled triggers...etc
+    @Test
+    void failedEvaluation2() {
```

Comment:
Please name the test more accuratly, like `failedExecutionFromInvalidExpression()`.